src/examples/forms/range/RangeValidation.tsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import { Field, Label, Range, Message } from '@zendeskgarden/react-forms';
+import { Row, Col } from '@zendeskgarden/react-grid';
+
+const valType: Record<number, any> = {
+  0: {
+    validation: 'success',
+    message: 'Growing all the time'
+  },
+  1: {
+    validation: 'warning',
+    message: 'Growing reguarly'
+  },
+  2: {
+    validation: 'error',
+    message: 'Growing slowly'
+  }
+};
```

Comment:
I think restructuring the object is more suitable in this example. What do you think about using an object keyed by the validation type?

```tsx
type validationTypes = 'success' | 'warning' | 'error';

const messages: Record<validationTypes, string> = {
  error: 'Growing slowly',
  warning: 'Growing reguarly',
  success: 'Growing all the time'
};
```

Reply:
Ideally, the `Message` component can consume the data using a object lookup like this:

```tsx
<Message validation={validation}>{messages[validation]}</Message>
```